public/tablet.html

```diff
@@ -16,7 +16,17 @@
  */ -->
 <html>
 <head>
-	<meta name="viewport" content="">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0 , maximum-scale=1.0, user-scalable=0">
+	<link rel="manifest" href="/manifest.json">
+	<!-- PWA support -->
+	<meta name="theme-color" content="#000000"/>
+	<!-- ios support -->
+	<link rel="apple-touch-icon" href="/img/brand/icons-72-square.png">
+	<link rel="apple-touch-icon" href="/img/brand/icons-96-square.png">
+	<link rel="apple-touch-icon" href="/img/brand/icons-128-square.png">
+	<link rel="apple-touch-icon" href="/img/brand/icons-512-square.png">
```

Comment:
You should add `sizes="512x512"` if you're defining `apple-touch-icon` multiple times

Reply:
Done :) 8b3167fd0e294528606d75d2a22f46c05062c123